Makefile

```diff
@@ -0,0 +1,17 @@
+.PHONY: up build down dependencies test
+
+down:
+	docker-compose down
+
+build:
+	docker-compose build
+
+up: build
+	docker-compose up -d
```

Comment:
I prefer it without the `-d` flag so you can see the logs. Any chance we can make that flag optional?

Reply:
We can have a different target for logs. Does that make sense?

Reply:
Sure